content/courses/native-onchain-development/program-state-management.md

```diff
@@ -554,11 +554,10 @@ We're ready to build and deploy our program!
 You can test your program by submitting a transaction with the right instruction
 data. For that, feel free to use
 [this script](https://github.com/Unboxed-Software/solana-movie-client) or
-[the frontend](https://github.com/Unboxed-Software/solana-movie-frontend) we
-built in the
-[Deserialize Custom Instruction Data lesson](deserialize-custom-data). In both
-cases, make sure you copy and paste the program ID for your program into the
-appropriate area of the source code to make sure you're testing the right
+[the frontend](https://github.com/EmekaManuel/movie-review-dapp) we built in the
+[Deserialize Custom Instruction Data lesson](/content/courses/native-onchain-development/deserialize-custom-data-frontend.md).
+In both cases, make sure you copy and paste the program ID for your program into
+the appropriate area of the source code to make sure you're testing the right
```

Comment:
Name the specific file.

Reply:
PS. Looking elsewhere, if the program ID is in two places, make it be in one place rather than duplicate the code.

Reply:
> PS. Looking elsewhere, if the program ID is in two places, make it be in one place rather than duplicate the code.

fixed ✅

Reply:
I can still see "make sure you copy and paste the program ID for your program into the appropriate area of the source code" above - there is no information about what file and what line to modify.

Reply:
Oh wait I can see it's below. I'll make a suggestion to make this more obvious.